File: pubsub/src/main/java/glitch/pubsub/events/ReconnectAlertEvent.java
package glitch.pubsub.events;

import glitch.api.ws.events.AbstractEvent;
import glitch.api.ws.events.IEvent;
import glitch.pubsub.GlitchPubSub;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class ReconnectAlertEvent extends AbstractEvent<GlitchPubSub> implements IEvent<GlitchPubSub> {
    public ReconnectAlertEvent(GlitchPubSub client) {
        super(client);
    }
}

File: pubsub/src/main/kotlin/glitch/pubsub/events/ReconnectAlertEvent.kt
package glitch.pubsub.events

import glitch.api.ws.events.AbstractEvent
import glitch.api.ws.events.IEvent
import glitch.pubsub.GlitchPubSub

class ReconnectAlertEvent(client: GlitchPubSub) : AbstractEvent<GlitchPubSub>(client), IEvent<GlitchPubSub> {
}
